getting-started/002-search.md

```diff
@@ -37,20 +35,23 @@ If you clicked on the query link above, you probably saw some cool **GeoJSON**,
 * YMCA, Jefferson, OH
 * YMCA, Belleville, IL
 
-Note that the results are spread out throughout the world. Since we haven't told the service anything about our current location or any other geographic context.
+Spelling matters, but not capitalization when performing a query with Mapzen Search. You can type `ymca`, `YMCA`, or even `yMcA`. See for yourself by comparing the results of the previous search to the following:
```

Comment:
agreed, maybe refactor this to a section dedicated to linguistic analysis techniques and how the effect results matching.